.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
     "printWidth": 100,
     "singleQuote": true,
-    "tabWidth": 4
+    "tabWidth": 4,
+    "trailingComma": "es5"
```

Comment:
Many files changed with the new behavior. Can enable the default behavior of prettier@v3.0 in a separate PR.